2019/python/lar_constraints.py

```diff
@@ -994,9 +994,10 @@ def v696_const(self, row):
 			row["aus_code_16"] = ""
 
 		#number of reported systems must match the number of reported results
-		result_enums = ("1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14","15", "16")
+		result_enums = ("1111","1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14","15", "16", 
+			"17", "18", "19", "20", "21", "22", "23", "24", "")
```

Comment:
I think having a blank in the results enumeration set is not correct.

Choosing a blank from a valid enumerations set for a system result where a system is reported should result in an edit fail.

Reply:
Hi David - changes were made that were reverted by my last commit. I pushed new changes now.